keyboards/durgod/hades/iso/rules.mk

```diff
@@ -0,0 +1 @@
+DEFAULT_FOLDER=durgod/dgk6x/hades_iso
```

Comment:
Recommend mirroring this in `durgod/hades/ansi/rules.mk`. 

I have created a PR for this: https://github.com/ebastler/qmk_firmware/pull/1

Reply:
Oh, out of curiosity - did you get any files from durgod, or did you actually manage to reverse engineer the whole PCB from scratch? The ISO version only needed few changes, so it was mostly trial and error for me (and I happened to find the right way at first attempt), but reversing a whole per-key RGB PCB sounds really tough.

Reply:
No files from durgod; the initial version for K320 was based off from kuenhlee.  The Hades/Venus/Galaxy board is similar to the K320 board, but required tweaking timings and tracking down wiring.